modules/python/src2/cv2.cpp

```diff
@@ -1608,6 +1608,26 @@ template<typename _Tp> static inline bool pyopencv_to_generic_vec(PyObject* obj,
     return true;
 }
 
+template<>
+bool pyopencv_to_generic_vec<bool>(PyObject* obj, std::vector<bool>& value, const ArgInfo& info)
+{
+    if(!obj || obj == Py_None)
+       return true;
+    if (!PySequence_Check(obj))
+        return false;
+    size_t n = PySequence_Size(obj);
+    value.resize(n);
+    for(size_t i = 0; i < n; i++ )
+    {
+        SafeSeqItem item_wrap(obj, i);
+        bool elem;
```

Comment:
It makes sense to reuse generic implementation above instead of this specialization with minor modifications:

```
        _Tp elem;
        if(!pyopencv_to(item_wrap.item, elem, info))
            return false;
        value[i] = std::move(elem);
```

Reply:
Fixed

Reply:
Not fixed, sorry. Because not all structures in `opencv_contrib` has a default ctor, so it cause warnings.